src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -121,13 +121,6 @@ def __init__(
         steps_offset: int = 0,
         **kwargs,
     ):
-        deprecate(
-            "tensor_format",
-            "0.6.0",
-            "If you're running your code in PyTorch, you can safely remove this argument.",
-            take_from=kwargs,
-        )
-
```

Comment:
Removing these deprecations now, as they're fairly harmless.

Reply:
`**kwargs` has to be removed as well as it's now left unchecked and it was added only for backwards breaking comp which we're breaking now